mftf/2.3/merging.md

```diff
@@ -270,6 +278,35 @@ The `LogInAsAdminTest` result corresponds to:
 </test>
 ```
 
+## Merge action groups
+
+Merging action groups allows you to extend existing tests by reusing existing action groups, while customizing them for your specific needs.
+
+### Use Case
+
+Here is an action group for selecting `customerGroup` in the `Cart Price Rules` section.
+The controls change drastically in the B2B version, so it was abstracted to an action group.
```

Comment:
```suggestion
The controls change drastically in the B2B version, so it was abstracted to an action group so that it may be easily changed if B2B is enabled. 
```